sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=iamianmullins_twincam
+sonar.organization=iamianmullins
+
+# This is the name and version displayed in the SonarCloud UI.
```

Comment:
Could we remove everything here except for the top two lines please, want to keep it simple, less maintenance needed for these files then, twincam is on version 0.7, want to keep that number in a single place.